module5/projects/DemoGuide.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: page
+title: Demo Guide
+---
+
+A well developmed project domo is a great opportunity to showcase your hard work and to practice talking about a project in front of a group of people. Below is a guide on how to present a project.
+
+
+## Guidelines
+
+1. __Important:__ Start the presentation by actually demoing your app. It is recommended that you skip demoing the log in/log out functionality. You can mention it in the demo, but start your demo already logged in. This will help save you time to talk about the more interesting features. Plan to live demo your application, but have a recording ready just in case there are unexpected issues. The recording is a nice backup.
```

Comment:
I'm thinking we are adamant that they don't have any login/logout auth stuff as part of their MVP. Thoughts on including this in the MVP section above? And including something here about skipping demoing log in / log out if that is part of your app's functionality?

Reply:
Yep, I like that callout in the MVP!